test/verifiedTests/testSteadyCom/testSteadyCom.m

```diff
@@ -0,0 +1,186 @@
+indCom = infoCom2indCom(modelJoint);  % get indCom from infoCom
+assert(isequal(indCom, modelJoint.indCom))
+infoCom = infoCom2indCom(modelJoint, modelJoint.indCom, true, {'Org1'; 'Org2'});  % get infoCom from indCom
+assert(isequal(infoCom, modelJoint.infoCom))
+
+switch CBT_LP_SOLVER
+    case {'gurobi', 'ibm_cplex', 'glpk', 'dqqMinos', 'quadMinos'}
+        feasTol = 1e-8;  % feasibility tolerance
+        tol = 1e-3;  % tolerance for comparing results
+    case {'mosek', 'matlab'}
+        feasTol = 1e-6;  % feasibility tolerance
+        tol = 1e-3;  % tolerance for comparing results
+    otherwise
+        feasTol = 1e-4;  % feasibility tolerance
+        tol = 1e-2;  % tolerance for comparing results
+end
+
+% TEST SteadyCom
+options.algorithm = 2;
+options.GRtol = 1e-6;  % tolerance for max. growth rate gap
+[~, result] = SteadyCom(modelJoint, options, 'feasTol', feasTol);
+data = load('refData_SteadyCom', 'result');
+% only the maximum growth rate must be equal. Others may differ.
+assert(abs(result.GRmax - data.result.GRmax) < tol)
+
+% TEST SteadyComFVA
+options.optGRpercent = [100 90 80];
+options.rxnNameList = {'X_Org1'; 'X_Org2'};
+[minFlux, maxFlux, ~, ~, GRvector] = SteadyComFVA(modelJoint, options, 'feasTol', feasTol);
+data = load('refData_SteadyComFVA', 'minFlux', 'maxFlux', 'GRvector');
+
+% Different solvers may give slightly different results. Give a percentage tolerance
+assert(max(max(abs(minFlux - data.minFlux) ./ data.minFlux)) < tol)
+assert(max(max(abs(maxFlux - data.maxFlux) ./ data.maxFlux)) < tol)
+assert(max(abs(GRvector - data.GRvector) ./ data.GRvector) < tol)
+
+% TEST SteadyComPOA
+options.savePOA = ['testSteadyComPOA' filesep 'test'];
+% look at the relationship between the abundance of Org1 and its exchange of b and c
+options.rxnNameList = [{'X_Org1'}; modelJoint.infoCom.EXsp(bCom | cCom, 1)];
+options.Nstep = 25;
+[POAtable, fluxRange, Stat, GRvector] = SteadyComPOA(modelJoint, options, 'feasTol', feasTol);
+data = load('refData_SteadyComPOA', 'POAtable', 'fluxRange', 'Stat', 'GRvector');
+devPOA = 0;  % percentage deviation
+devSt = 0;  % absolute deviation of the correlation statistics (since zeros may appear here)
+for i = 1:size(POAtable, 1)
+    for j = 1:size(POAtable, 2)
+        if ~isempty(POAtable{i, j})
+            devPOA = max(devPOA, max(max(max(abs(POAtable{i, j} - data.POAtable{i, j}) ./ abs(data.POAtable{i, j})))));
+            devSt = max(devSt, max(abs(Stat(i, j).cor - data.Stat(i, j).cor)));
+            devSt = max(devSt, max(abs(Stat(i, j).r2 - data.Stat(i, j).r2)));
+        end
+    end
+end
+assert(devPOA < tol)
+assert(max(max(max(abs(fluxRange - data.fluxRange) ./ abs(data.fluxRange)))) < tol)
+assert(devSt < tol)
+assert(max(abs(GRvector - data.GRvector) ./ data.GRvector) < tol)
+% delete created files
+rmdir([pwd filesep 'testSteadyComPOA'], 's')
+clear CBT_LP_SOLVER
+
+%%%%%% the section is only to avoid using ddqMinos. Should be deleted after it is fixed.
```

Comment:
@shjchan, on some versions of MATLAB, using `%%` in `runtests()` causes issues. May you replace these by simple comments `%`? Thanks.